example/README.md

```diff
@@ -0,0 +1,12 @@
+# ZKSyncSDK for Go usage example
+
+This simple test project show example of usage of imported package.
+
+## Quick run
+
+For macOS and Linux just run make: 
```

Comment:
Will it work for Windows?

Reply:
This `makefile` was designed for Mac and Linux only, to run example fast and easy.
Windows users must manually download appropriate zksync-crypto-c library for Windows and link it during compilation/execution (probably, it also will require some changes in example source code, I am not sure about Win particularities)

Reply:
It's ok for first release, but we should add tutorial for WIndows users as well in the future:
https://linear.app/matterlabs/issue/ZSDK-58/windows-tutorial-for-go-sdk